### DISCUSSION TITLE
New feature proposal: Sciences rating

### DISCUSSION THREAD

**Discussion by dvincent56** [Keriew/augustus#289] [Ideas]
I don't know if this new proposal is implementable.
But I propose these new feature to add.
Value of bonus are for information and open to discussion.

**Sciences**

Adding new ratings (sciences)
Sciences rating increase other ratings by step (0-25 => +2 / 25-50 => +4 / 50-75 => +8 / 75-100 => +10)
Adding new buildings to increase sciences ratings
These new buildings require other buildings to be build (chaining building) and are disabled if the required building is destroy.

Vanilla buildings (Tier 1):
- School (science rating +2)
- Library (science rating +2)
- University (science rating +3)
- Academy (science rating +3)

New buildings :
Tier 2 :
- Laboratory <require School> / increase teacher speed (science rating +5)
- Scriptonium <require Library> / increase bookseller speed (science rating +5)
- Observatory <require University> / increase trading boat speed (science rating +5)
- Study <require Academy> / increase firefighters and Engineers speed (science rating +5)

Tier 3 :
- Apothecary <require Laboratory> / increase significantly the health rating (science rating +10) 
- Horologium < require Observatory> / increase significantly the culture rating (science rating +10)

Tier 4 :
- Scientists guild <require a sciences rating more than 75> / increase significantly sciences rating but decrease the gods rating.

Moreover I propose a new Cultural building (bonus) :
- Odeon -> require Singing school
- Singing school -> provide new figure (singer/poet)

Waiting any feedback.
Regards,

**Comment by dvincent56** [comment]
I started an implementation.
Sciences is currently calculated from school/university/library coverage

I modify the Ratings interface like that.
![sciences](https://user-images.githubusercontent.com/81165266/112553723-517aad00-8dc5-11eb-8059-b767af69b1c0.png)

I will keep you posted on my progress

**Comment by Areldir** [comment]
If I could make a suggestion, could it be calculated from buildings which don't already count towards an existing rating (ie. schools, libraries, etc.)? Otherwise, its basically just the culture rating with another name.
Edit: also, to follow up on that, if you're planning on making this a PR for Augustus, could you please join us on discord to discuss the idea further?

**Reply by dvincent56** [reply]
I'm on discord aka awatibala.
Yes currently sciences is like culture. But I would like to remove these 3 buildings in culture calculation.

**Reply by maoxp09** [reply]
> 我和阿瓦提巴拉意见不一致。是的，目前科学就像文化。不过，我想删除这3幢建筑的文化计算。

This is a good suggestion. In fact, I think with you. What's more is the scientific evaluation. I put forward some new ideas. The number of school libraries and universities can only be increased by 10, otherwise it will overlap with culture. Scientific rating should be mainly related to the climbing and upgrading of science and technology tree!

**Reply by maoxp09** [reply]
May I ask how to join your discord

**Reply by MSTVD** [reply]
Shoot me a message on discord, username @mstvd

**Comment by maoxp09** [comment]
This is a good suggestion. In fact, I think with you. What's more is the scientific evaluation. I put forward some new ideas. The number of school libraries and universities can only be increased by 10, otherwise it will overlap with culture. Scientific rating should be related to the level of science and technology tree

**Comment by maoxp09** [comment]
I'm going to propose to add the function of science and technology tree, which is divided into five categories: military culture, economy, politics, science and technology. Upgrading skills requires consumption of goods or money plus buff, such as increasing housing capacity, industrial and agricultural output, catapult firing speed, residents' happiness, education coverage, increasing warehouse capacity, etc. I'm going to make some design drawings for your university library Only 10 more scientific evaluation will be added, and the additional 90 will need to be upgraded by the level of science and technology tree

**Comment by Mephisto-Fibs** [comment]
This would be very unroman and would unnecessarily complicate the gaming experience. Caesar 3 is not a game in the cililization series; if you want a techtree, then play one of the numerous games that has one. 

If any additional element is to be added, then this should be either a religious or a political one. 

**Religion:** 
The Roman year was constituted of numerous festivals for countless deities, and the Romans celebrated these with devotion and earnestness. So far, this has been more or less well covered by the culture rating; if differentiation is supposed to refine the game, it should be done at this point of reference. (maybe by adding religious buildings that unlock the possibility for blessings from the gods... which is reguated in the game options for difficulty at this point) 


**Politics:** 
The politics in Rome, with its strictly hierarchical structure of social order, has not been represented at all by the game so far. Yes, the ranking of a title for the governor has been implemented, but so far this only regulates the salary of the governor, not his political power. A governor's powers and authority could be calculated depending on how high the player is in rank and the census in a city. 
This could result in the Senate building playing a larger and more useful role, by allowing it to directly affect city management options. 
By now, these options for city management have to be clicked on and off by the player in the game options menue. What I suggest here, is to implement these menue options into the process of the game by bringing certain city management options to the senate building. The governor's power in the Senate could be calculated by census and rank; the higher the census rating is, the more options the player gets to choose from. These three Options are: all houses merge, allow global labour pool and retirement age at 60 instead of 50.  

What do you think?

**Reply by dmyersturnbull** [reply]
Small note: No comment here about the overall idea, but: Global labor pool massively changes the game, so I don't think it should be researchable like that; the same would apply for things like walker distances. And being able to toggle retirement age by +10 seems like a very powerful tool.

**Reply by dmyersturnbull** [reply]
I think I like the idea of a few researchable items, either via science or politics, especially if some are military (e.g. increase ballista speed). C3's military system is so bad that I always felt it should be either removed or dramatically changed. Maybe political research could enable more (hopefully less broken) unit types like archers, or make walls stronger so you don't have to stack walls to 8 depth.

**Comment by tmancini85** [comment]
About politics factor, I wrote some proposals (involving Senate and governor's residences) in https://github.com/Keriew/augustus/discussions/425#discussioncomment-7998200.
